debezium-server/debezium-server-bom/pom.xml

```diff
@@ -203,13 +208,24 @@
                 <artifactId>pravega-client</artifactId>
                 <version>${version.pravega}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.rocketmq</groupId>
+                <artifactId>rocketmq-client</artifactId>
+                <version>${version.rocketmq}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.rocketmq</groupId>
+                <artifactId>rocketmq-tools</artifactId>
+                <version>${version.rocketmq}</version>
```

Comment:
IMHO you need to put exclusion of `commons-cli` here - there is a clash with quarkus dependencies
```
                <exclusions>
                    <exclusion>
                        <groupId>commons-cli</groupId>
                        <artifactId>commons-cli</artifactId>
                    </exclusion>
                </exclusions>
```